addons/website/views/snippets/s_blockquote.xml

```diff
@@ -18,45 +22,60 @@
     <xpath expr="." position="inside">
         <!-- Layout -->
         <div data-js="Blockquote" data-selector=".s_blockquote">
-            <we-select string="Display">
-                <we-button data-select-class="s_blockquote_classic" data-display="classic">Classic</we-button>
-                <we-button data-select-class="s_blockquote_cover" data-display="cover" data-name="blockquote_cover_opt">Cover</we-button>
-                <we-button data-select-class="s_blockquote_minimalist" data-display="minimalist" data-name="blockquote_minimalist_opt">Minimalist</we-button>
+            <we-select string="Style">
```

Comment:
These "Styles" don't really provide a lot of variations and 90% of the changes can be easily achieved by using the editor (see bg-color, text alignment...).
I propose to streamline the option by reducing it to what's really "unique" for each of these design.
What about renaming it to "Decornation" and provide the following options:
- None
- Line
- Icon
- Line and Icon

?

The only missing point would be the `s_blockquote_infos` alignment. This can be easy achieved by using a dedicated controller. Indeed it makes sense to be able to change it regardless by other options.

Something like this (to be tested):
```xml
<div  data-selector=". s_blockquote" data-target=".s_blockquote_infos">
  <we-select string="Alignment">
      <we-button data-select-class="flex-row justify-content-start">Left</we-button>
      <we-button data-select-class="flex-row-reverse justify-content-end">Right</we-button>
      <we-button data-select-class="flex-columns align-items-center">Center</we-button>
  [...]
 ```